flyway-community-db-support/flyway-database-clickhouse/src/main/java/org/flywaydb/community/database/clickhouse/ClickHouseConfigurationExtension.java

```diff
@@ -23,8 +23,10 @@
 @Getter
 public class ClickHouseConfigurationExtension implements ConfigurationExtension {
     private static final String CLUSTER_NAME = "flyway.clickhouse.clusterName";
+    private static final String ZOOKEEPER_PATH = "flyway.zookeeper.path";
```

Comment:
Let's change the constant to "flyway.clickhouse.zookeeperPath" besause it applies to ClickHouse only.